### ISSUE TITLE
OCtober Cms startic page plugin

### ISSUE THREAD

**Issue by durdymyratahmedow** [octobercms/october#5607]
Hi,

I am following the tutorial "Getting Started with Static Pages" (http://octobercms.com/blog/post/getting-started-static-pages), when adding the menu item for the Blog Categories there are no results found for CMS Pages.

These are the steps I have followed:

Add menu - name: Blog Categories
Clicked "Add Item"
Select Type "All blog categories"
Checked "Replace this item with generated children"
Select "CMS Page" - there are no pages listed in the dropdown, and displays "No results found"
If I select Type "CMS Page" all the pages are listed in the dropdown, however if I select "Blog Category" or "All blog categories" then the CMS Pages are empty in the dropdown list.

Any assistance would be appreciated.

Thanks

**Comment by daftspunk** [comment]
Hi @durdymyratahmedow!

The dropdown list expects to find a CMS page with the Blog Posts component that uses a `categoryFilter` specifying a route parameter (eg: `{{ :slug }}`)

```
[blogPosts]
pageNumber = "{{ :page }}"
categoryFilter = "{{ :slug }}"
postsPerPage = 10
noPostsMessage = "No posts found"
sortOrder = "published_at desc"
categoryPage = 404
postPage = 404
```

Assuming you have this, the CMS page should appear. If it doesn't, it may be caused by a stale cache. Please run this CLI command to clear it

```
php artisan cache:clear
```

With the above conditions met, the page should appear in the menu list.

The stale cache problem is a bug that we will fix in the next release of October CMS. Thanks for reporting it. For a faster response time, you may email bug reports with the platform to hello@octobercms.com

**Comment by daftspunk** [comment]
Should be fixed in v2.0.16+ released today. Thanks!
